sensor/AdblockPlugin.js

```diff
@@ -78,7 +80,16 @@ class AdblockPlugin extends Sensor {
 
     async apiRun() {
       extensionManager.onSet("adblockConfig", async (msg, data) => {
```

Comment:
Need onGet function too for app to get current configuration. (default+redis configured)

Reply:
* what's configured by default in the config.json
* what's the runtime config that is configured by app in the redis

Reply:
so that the list of adblock lists can be displayed in the app

Reply:
ignore the above comments for now